src/main/app/claims/models/claim.ts

```diff
@@ -222,6 +234,29 @@ export class Claim {
     return this.settlement && !!this.settlementReachedAt
   }
 
+  private isCCJPaidWithinMonth (): boolean {
+    let futureMonth
+
+    if (this.countyCourtJudgmentIssuedAt) {
+      futureMonth = calculateMonthIncrement(this.countyCourtJudgmentIssuedAt)
+    } else if (this.countyCourtJudgmentRequestedAt) {
+      futureMonth = calculateMonthIncrement(this.countyCourtJudgmentRequestedAt)
+    }
+    return this.moneyReceivedOn.isSameOrBefore(futureMonth)
+  }
+
+  private hasCCJ (): boolean {
```

Comment:
as above. this can be removed. and put the check inside the above month

Reply:
if this function only checks one thing then just use that directly if your if statements. this function doesn't need to exist